ignite/contrib/handlers/mlflow_logger.py

```diff
@@ -258,3 +262,9 @@ def close(self):
         import mlflow
 
         mlflow.end_run()
+
+    def attach_output_handler(self, engine: Engine, event_name: str, *args: Any, **kwargs: Mapping):
```

Comment:
@sdesrozis @erip do you think we could refactor this part of code into base logger and avoid rewriting the same lines for all supported loggers ?

Reply:
I have been thinking about that. I don't know if it's possible, actually. 😦 My thought is that maybe there's some way to use inner classes to abstract over a specific implementation of `OutputHandler` and `OptimizerParamsHandler` ... but I don't think it's possible. See [here](https://stackoverflow.com/questions/61582840/is-there-some-mechanism-to-enforce-nested-interfaces-are-implemented-in-abstract).

Reply:
It may be possible with some incredible hackiness... maybe through registration or metaclass magic... we could define properties which are the `cls` constructor for the implementations specific to a given `BaseLogger` implementation. Not sure how to do this least invasively and in the least amount of hacks. 😆

Reply:
So let’s keep it simple ! 😉